data/remnant/remnant jobs.txt

```diff
@@ -307,14 +309,15 @@ mission "Remnant: Logistics Outfitter Delivery"
 		dialog phrase "generic cargo on visit"
 	on complete
 		dialog `A repair technician inspects the <cargo> parts and authorizes your payment of <payment>.`
-		payment 30000 25
+		payment
+		payment 30000
 
 
 
 mission "Remnant: Logistics Salvage Delivery"
 	name "Salvage delivery to <planet>"
 	description `Deliver <cargo> components from recent salvage operations to the outfitters on <planet> for research, refurbishment, or recycling. Logistics adjustment is <payment>.`
-	cargo "remnant salvage" 5 100
+	cargo "remnant salvage" 5 4 .08
```

Comment:
Average cargo amount of 63. About 15% of the time you'll find one with over 100 tons of cargo.
```suggestion
	cargo "remnant salvage" 5 4 .06
```